src/Psr/CacheItemPool/CacheItem.php

```diff
@@ -116,7 +116,7 @@ public function set($value): CacheItemInterface
     /**
      * {@inheritdoc}
      */
-    public function expiresAt($expiration): CacheItemInterface
+    public function expiresAt($expiration): static
```

Comment:
Can we also add argument types so we are aligned with the interface?

Reply:
added (needed to drop "psr/cache": "^1.0  for it to work)